homeworks/franktoth/2_vars_datatypes/fictive_character.py

```diff
@@ -0,0 +1,14 @@
+name = input("Enter your name: ").strip().capitalize()
+age = int(input("Enter your age: "))
+python_exp_in_years = int(input("How many years have you been using Python: "))
+user_prof_dev = input("Do you want to be a professional Python developer? (yes/no)")
+
+db_age_in_days = age*365        #calculation of days without leap years
+
+prof_dev_decision = "He/She wants to be a prof Python Developer" if user_prof_dev=="yes" else "He/She doesn't want to be a prof developer"
```

Comment:
Ez előbb elkerülte a figyelmem, most is csak FYI:
Ha a prof_dev_decision értéke csak annyi lenne hogy "wants" vagy "doesn't want", a többi szöveget be tudnád rakni a print-be, és csak ezeket kellene beágyazni, rövidítve a ternary-t.